contracts/misc/MerkleProof.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+library MerkleProof {
```

Comment:
I prefer to import library from here https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/MerkleProof.sol